sunpy/net/hek/hek.py

```diff
@@ -33,6 +33,19 @@ def _freeze(obj):
     return obj
 
 
+def convert_time(response_obj):
+    """
+    Converts time from str to astropy.time.core.Time for a HEKTable.
+    """
```

Comment:
I think this docstring needs expanding to have a parameters section, also a note to say this does the conversion in place. 

Maybe we want it to return a new repesonse?